hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2988,4 +2988,13 @@
       OM/SCM/DN/S3GATEWAY Server connection timeout in milliseconds.
     </description>
   </property>
+
+  <property>
+    <name>ozone.default.bucket.layout</name>
+    <value>OBS</value>
```

Comment:
Please use full name `OBJECT_STORE` instead of OBS.

[BucketLayout.java#L30](https://github.com/apache/ozone/blob/master/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/BucketLayout.java#L30)